### ISSUE TITLE
Harmonize District Heating/District Heat names

### ISSUE THREAD

**Issue by zwiebo** [netzero2040/netzero2040#30]
# District heating is called District heat in
- Secondary Energy|District Heat:
- Conversion Input|District Heat|{District Heat Input}
etc. 

https://github.com/netzero2040/netzero2040/blob/main/definitions/variable/energy.yaml

do we need to change this?

**Comment by danielhuppmann** [comment]
Yes if you want to know how much District Heat was generated from each fuel/input...
